radix-engine/src/updates/cuttlefish.rs

```diff
@@ -89,6 +96,12 @@ fn generate_principal_batch(
             generate_system_logic_v2_updates(store),
         ));
     }
+    if let UpdateSetting::Enabled(_settings) = &always_visible_global_nodes_update {
+        transactions.push(ProtocolUpdateTransactionDetails::flash(
+            "cuttlefish-protocol-update-always-visible-global-nodes",
```

Comment:
Ditto on the naming scheme. (Unless we choose to separate the two updates into two, to selectively enable them)

Reply:
Yup, naming sounds good to me. Will go with the name you recommended.